app/src/main/java/com/android/solvit/MainActivity.kt

```diff
@@ -132,10 +132,17 @@ fun SeekerUI(
     }
     composable(Route.MESSAGE) { MessageScreen(navigationActions) }
     composable(Route.CREATE_REQUEST) {
-      CreateRequestScreen(navigationActions, serviceRequestViewModel)
+      user?.let { user ->
+        CreateRequestScreen(
+            navigationActions = navigationActions,
+            requestViewModel = serviceRequestViewModel,
+            userId = user.uid)
+      }
```

Comment:
I think that instead of using user.uid, it's better to directly use Firebase.auth.currentUser.uid

Reply:
Thank you for your advice, I have applied the recommended modification in my latest commit.